utils/releaser/src/ReleaseWorker/Release/CheckChangelogForTodaysDateReleaseWorker.php

```diff
@@ -82,6 +83,7 @@ public function work(Version $version): void
             ));
 
             $this->commit('CHANGELOG.md date updated to today');
+            $this->processRunner->run('git push');
```

Comment:
Can you please update printed note (`CHANGELOG.md date for "%s" version was updated to "%s".`) and worker description so it's clear this worker pushes?

Reply:
New opinion, please keep pushes manual :) Right now I need to test something and pushed branch by mistake.

Reply:
updated

Reply:
fixed